docs/instrumentation-libraries.md

```diff
@@ -0,0 +1,117 @@
+| [OWIN](https://docs.microsoft.com/en-us/aspnet/aspnet/overview/owin-and-katana/) | | | This is .NET Framework only. |
+
+### Databases
+
+| Library | [Tracing Support](./config.md#instrumented-traces-libraries-and-frameworks) | [Metrics Support](./config.md#instrumented-metrics-libraries-and-frameworks)| Databases Tested | Notes |
+| --- | :---: | :---: | --- | --- |
+| [Entity Framework](https://docs.microsoft.com/en-us/ef/ef6/) | | | | Needs investigation. It might be implicitly supported based on the configured [database provider](https://docs.microsoft.com/en-us/ef/ef6/fundamentals/providers/). |
+| [Entity Framework Core](https://docs.microsoft.com/en-us/ef/core/) | | | | Needs investigation. It might be implicitly supported based on the configured [database provider](https://docs.microsoft.com/en-us/ef/core/providers/). |
+| [Dapper](https://github.com/DapperLib/Dapper) | | | | Needs investigation. It might be implicitly supported based on the configured [database provider](https://github.com/DapperLib/Dapper#will-dapper-work-with-my-db-provider). |
+| [Microsoft.Data.SqlClient](https://www.nuget.org/packages/Microsoft.Data.SqlClient) | Yes | | [MS SQL Server](https://www.microsoft.com/en-us/sql-server/sql-server-2019) | |
+| [System.Data.SqlClient](https://www.nuget.org/packages/System.Data.SqlClient) | Yes | | | |
+| [MySql.Data](https://dev.mysql.com/doc/connector-net/en/connector-net-introduction.html) | | | | This is the official [MySQL](https://dev.mysql.com/) library. |
+| [MySqlConnector](https://mysqlconnector.net/) | | | | Seems to be the [recommended library for MariaDB](https://mariadb.com/kb/en/mysqlconnector-for-adonet/). |
+| [Npgsql](https://www.npgsql.org/) | Yes | | [PostgreSQL](https://www.postgresql.org/) | |
+| [Microsoft.Data.SqlLite](https://docs.microsoft.com/en-us/dotnet/standard/data/sqlite/?tabs=netcore-cli) | | | | |
+| [MongoDB.Driver](https://www.nuget.org/packages/mongodb.driver) | Yes | | [MongoDB](https://www.mongodb.com/docs/) | |
+| [StackExchange.Redis](https://stackexchange.github.io/StackExchange.Redis/) | | | | |
+| [Elasticsearch-net](https://github.com/elastic/elasticsearch-net) | | | | We should be able to use [OpenTelemetry.Instrumentation.ElasticsearchClient](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Instrumentation.ElasticsearchClient). |
+| [Oracle.ManagedDataAccess.Core](https://www.nuget.org/packages/Oracle.ManagedDataAccess.Core) | | | | |
+| [Oracle.ManagedDataAccess](https://www.nuget.org/packages/Oracle.ManagedDataAccess) | | | | |
+| [AWSSDK.DynamoDBv2](https://www.nuget.org/packages/AWSSDK.DynamoDBv2) | | | | |
+| [Microsoft.Azure.Cosmos](https://www.nuget.org/packages/Microsoft.Azure.Cosmos) | | | | |
+
+### External request libraries (REST/RPC)
+
+1. HttpClient
+1. HttpWebRequest
+1. WCF
+1. RestSharp
+1. Grpc
+1. GraphQL
+1. WCF Core
+
+### Messaging
+
+1. RabbitMQ
+1. MSMQ
+1. Kafka
+1. NServiceBus
```

Comment:
Should we add here the open source competitor also: [MassTransit](https://www.nuget.org/packages/MassTransit/8.0.6-develop.512)

Reply:
It looks like nuget usage is high enough, and there is an instrumentation library already.